plugins/UsersManager/vue/src/PagedUsersList/PagedUsersList.vue

```diff
@@ -277,104 +294,105 @@
                   :disabled="user.role === 'superuser'"
                   uicontrol="select"
                   :options="user.login !== 'anonymous' ? accessLevels : anonymousAccessLevels"
-                />
-              </div>
-            </td>
-            <td
+              />
+            </div>
+          </td>
+          <td
               id="email"
               v-if="currentUserRole === 'superuser'"
-            >{{ user.email }}</td>
-            <td
+          >{{ user.email }}
+          </td>
+          <td
               id="twofa"
               v-if="currentUserRole === 'superuser'"
-            >
+          >
               <span
-                class="icon-ok"
-                v-if="user.uses_2fa"
+                  class="icon-ok"
+                  v-if="user.uses_2fa"
               />
-              <span
+            <span
                 class="icon-close"
                 v-if="!user.uses_2fa"
-              />
-            </td>
-            <td
+            />
+          </td>
+          <td
               id="last_seen"
               v-if="currentUserRole === 'superuser'"
-            >
-              {{ user.last_seen ? `${user.last_seen} ago` : '-' }}
-            </td>
-            <td id="status">
-              <span :class="user.invite_status">
+          >
+            {{ user.last_seen ? `${user.last_seen} ago`:'-' }}
+          </td>
+          <td id="status">
+              <span :class="Number.isInteger(user.invite_status)? 'pending':user.invite_status">
               {{ getInviteStatus(user.invite_status) }}
                 </span>
-            </td>
-            <td class="center actions-cell">
-              <button
-                  class="resend table-action"
-                  title="Resend Invite"
-                  @click="userToChange = user; showResendConfirm()"
-                  v-if="user.invite_status!=='accept'"
-              >
-                <span class="icon-email" />
-              </button>
+          </td>
+          <td class="center actions-cell">
+            <button
+                class="resend table-action"
+                title="Resend Invite"
+                @click="userToChange = user; showResendConfirm()"
+                v-if="user.invite_status!=='active' && user.invite_status!=='declined'"
+            >
+              <span class="icon-email"/>
+            </button>
 
-              <button
+            <button
                 class="edituser table-action"
                 title="Edit"
                 @click="$emit('editUser', { user: user })"
-                v-if="user.login !== 'anonymous'"
-              >
-                <span class="icon-edit" />
-              </button>
-              <button
+                v-if="user.login !== 'anonymous' && user.invite_status!=='declined'"
+            >
+              <span class="icon-edit"/>
+            </button>
```

Comment:
Currently only the superuser can see the delete button below. But for invited users also the user who invited someone should be able to delete the user / cancel the invite.